Nitean recognition.md

```diff
@@ -0,0 +1,27 @@
+# Nitean recognition
+
+As Niteans, we believe in celebrating small wins and recognizing achievements. By enabling Nitean recognition, we're boosting each other's morale, enhancing engagement and communication, and improving the overall company culture.
+
+## Wall of Happy
+
+This is inspired by head of Google’s People Operations, Laszlo Bock. He has a Wall of Happy outside his office where kudos are printed and put on a physical wall.
```

Comment:
Do we have a link to show this?

Reply:
There's an image: https://lh3.googleusercontent.com/-4ksu7hCKgfQ/VR6TJ3OxvOI/AAAAAAABZIQ/8GS-3RHNKY4/w1008-h720/WR%2B-%2BPhoto%2B-%2BWall%2Bof%2BHappy%2B-%2BG%252B%252C%2BLI%252C%2BTwitter.jpg

Reply:
Mhm. I already linked that.